sherpa/astro/instrument.py

```diff
@@ -1288,6 +1288,10 @@ def create_delta_rmf(rmflo, rmfhi, offset=1,
 
     .. versionadded:: 4.10.1
 
+    .. versionchanged:: 4.13.2
+
+       Added header keyword
+
```

Comment:
Do we really need a "versionchanged" for adding a keywords that won't usually be used? I put it here because that's where other functions have it, but personally I feel that `versionchanged` can be reserved for breaking changes; we don't need to keep a changelog of each function in the docs. For things that are as small as this one we have the changelog. Let me know if you agree, because I'd be happy to remove this.

Reply:
I have "complicated" thoughts on issues like this because combing through multiple changelogs to find out when something was changed is not a great experience. However, there's plenty of changes I'm making that are more significant than adding a keyword to a rarely-used function that never get called out (because I forgot to add it or the behavior is from combination of things so a versionchanged line isn't useful to add!). On the other hand, having five or more sections listing things that have changed does not improve the documentation experience! What I would like to discuss at some point is #555 but that's not relevant for this PR.

I don't believe there's a way to come up with a hard-and-fast rule for when something like this is needed or not (e.g. things like removing a parameter or changing some fundamental behavior is more important in general than adding an optional argument, but if that optional argument adds new functionality to an often-requested function then ...). So we have to go with our guts.

I think in this case the versionchanged` line is not needed.

Reply:
done.